core/src/components/avatar/avatar.common.scss

```diff
@@ -25,3 +25,10 @@
 
   overflow: hidden;
 }
+
+// Disabled Checkbox
+// ---------------------------------------------
+
+:host(.avatar-disabled) {
```

Comment:
Why do we add this here when there are no styles for the `md` and `ios` themes?

Reply:
I thought it was so generic it makes sense there, but I'll put it on ionic.scss